.github/workflows/mingw.yml

```diff
@@ -35,10 +40,11 @@ jobs:
         with:
           path: src
       - name: Set up Ruby & MSYS2
-        uses: MSP-Greg/setup-ruby-pkgs@v1
+        uses: MSP-Greg/setup-ruby-pkgs@ucrt
         with:
-          ruby-version: 2.6
-          mingw: _upgrade_ gmp libffi libyaml openssl ragel readline
+          ruby-version: ${{ matrix.base_ruby }}
+          setup-ruby-ref: MSP-Greg/ruby-setup-ruby/00-win-ucrt
```

Comment:
This line is warned as an invalid input.
```
Unexpected input(s) 'setup-ruby-ref', valid inputs are ['ruby-version', 'bundler', 'bundler-cache', 'working-directory', 'apt-get', 'brew', 'mingw', 'msys2', 'mswin', 'choco', 'vcpkg']
```
```suggestion
```

Reply:
The warning is caused by a missing option declaration, but the option is currently necessary. I opened a PR to fix this: https://github.com/MSP-Greg/setup-ruby-pkgs/pull/17

Reply:
I see.
It’s ok if this option is just warned but effective.

Reply:
We had some discussions about the right way to implement UCRT support, which aren't finalized yet. Now the latest Github Action image changed things again, so that I guess it will take more weeks. It was also a bit of chicken-egg problem, to add UCRT support before it's supported by Ruby core.

I could add all necessary commands to this `mingw.yml` file manually, but this makes it harder to follow, when the UCRT support was merged to regular ruby-setup.